CSS/syntax_test_css.css

```diff
@@ -4479,7 +4479,7 @@ img{
 /*               ^ variable.language.parent.css */
 
         /* & is not a string value, but an object in CSS */
-        .foo { &bar { /* is not not .foobar!  */ }
+        .foo { &bar { /* is not .foobar!  */ }
```

Comment:
this is notable.

Reply:
This one does not seem that important, but still an improvement.